<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Dragon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 255, 0);
background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>DRAGON: G</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Arabesco">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 251, 0);
border-color: rgb(255, 251, 0);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>ARABESCO: A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Henon">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 228);
border-color: rgb(255, 0, 228);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>HENON: H</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Malthus1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(107, 255, 246);
border-color: rgb(107, 255, 246);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>MALTHUS 1: M</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Malthus2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>550</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 213, 195);
border-color: rgb(241, 213, 195);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>MALTHUS 2: N</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Detener">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>550</y>
      <width>104</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);
border-color: rgb(85, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>DETENER: D</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Salir">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>550</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
border-color: rgb(170, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>SALIR: S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mandel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>550</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(112, 11, 200);
border-color: rgb(112, 11, 200);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>MANDEL: E</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
